### DISCUSSION TITLE
Buttons in Table

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#4289] [Help] [answered]
Hello,

is there a way to put a "-" before and a "+" after the amount column to reduce or increase the amount?
![Screenshot 2022-02-02 085941](https://user-images.githubusercontent.com/95284278/152115267-8e101923-c7bd-4e46-ba3e-566d10ea6f2b.png)

**Comment by mjashanks** [comment, marked as answer]
Hi @kasgithub8 you cannot do this currently. However, keep an eye out for this issue: https://github.com/Budibase/budibase/issues/4200 which we plan to release at the end of February, along with other table improvements.

**Reply by kasgithub8** [reply]
Damn, thanks for the answer @mjashanks! So this will include buttons for columns too? I guess I could consider myself lucky then. 😄
